flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/PFD/A320_Neo_PFD.js

```diff
@@ -146,7 +142,7 @@ class A320_Neo_PFD_MainPage extends NavSystemPage {
             this.spdFlash.setAttribute("visibility", "visible");
             this.altFlash.setAttribute("visibility", "visible");
             this.vsFlash.setAttribute("visibility", "visible");
-        } else if (this.selfTestTimer > -5) {
```

Comment:
This condition was only ever `false` when the user had a framerate of 1 frame per 5 seconds or less. Therefore, I assumed this to be a coding error and removed the condition.